config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-presubmits.yaml

```diff
@@ -61,3 +61,13 @@ presubmits:
         hostPath:
           path: /sys/fs/cgroup
           type: Directory
+  - name: pull-cluster-api-provider-aws-verify-boilerplate
```

Comment:
This will also need an associated test-grid config, see https://github.com/kubernetes/test-infra/pull/10294 for an example